### DISCUSSION TITLE
MICR Code Recognizer

### DISCUSSION THREAD

**Discussion by Vadi** [microsoft/presidio#1482] [Ideas]
I'm trying to write a MICR Code recognizer. Not sure where to start. I thought may be I will use OpenCV but feels like it overkill. Any head start is greatly appreciated.
<img width="369" alt="image" src="https://github.com/user-attachments/assets/54e1ef79-966a-45e3-9373-734ea3293531">

**Comment by omri374** [comment]
Hi @Vadi. You mentioned OpenCV. Are you looking for MICR codes detection in images?
